src/main/java/org/polystat/Polystat.java

```diff
@@ -73,6 +81,45 @@ public static void main(final String... args) throws Exception {
         new Polystat(System.out).exec(args);
     }
 
+    private void logError(final String error) {
```

Comment:
@Sitiritis nobody is called these private methods. Why they are here?

Reply:
Forgot to remove the debug code, thanks for the notice